tests/test_sync.py

```diff
@@ -371,21 +389,7 @@ async def test_sync(
         generate_keys=False,
     )
 
-    # Set minimum publishers testing
```

Comment:
this test is redundant now (and was failing nondeterministically because of the assert on 376)